### ISSUE TITLE
Game crashed after I placed down a vehicle charge next to a level 3 charge pad

### ISSUE THREAD

**Issue by Bworms1980** [Cazadorsniper/PO3#948]
I placed down the vehicle charger next to a level 3 rocket pad from Extra Planets because I was about to build the electric rocket. From what I can remember, the pad still had the fuel loader and coal generator next to it, so I don't know if that may have caused the crash. Or if I put it next to the wrong launching pad. Please let me know what I can do to get my map working again. I pretty far into it and would hate to restart again. Thank you.

[crash-2020-03-25_07.18.00-server.txt](https://github.com/Cazadorsniper/PO3/files/4380534/crash-2020-03-25_07.18.00-server.txt)

**Comment by Bworms1980** [comment]
Sorry, I'm not sure how this works. It said to drag and drop the file, so I pulled the txt crash report here. Here's the Pastebin link, hope it works.

https://pastebin.com/Eq8QkmyE

**Comment by Cazadorsniper** [comment]
try updating

**Comment by Bworms1980** [comment]
Pack Version: 3.2.54

I updated and it lets me load up the game now, but when I try to open the map, it crashes. I'm up loading the new crash report. Don't know if its different from the last one.

https://pastebin.com/sbrL1p3Z

**Comment by Cazadorsniper** [comment]
There is an ender io tank causing the crash, at (-206,86,436) try removing it using mcedit or something like that or go to line 62 "B:removeErroringTileEntities=false" turn that to true start your world then set it back to false.

**Comment by Bworms1980** [comment]
I changed the removeerroring in the forge file, but when I went to open the map the game froze. I restarted and it froze when loading the map again. I loaded it up and tried to open the map again and while the loading words were up on the screen I went afk, came back 30 min later and it was still stuck on the loading screen. I decided to use my last backup from an hour before I placed the block and the game is working. I removed the tank you mentioned. Thank you.
